www/front_src/src/Resources/Graph/Performance/GraphActions.tsx

```diff
@@ -178,11 +179,12 @@ const GraphActions = ({
               data-testid={labelPerformanceGraphAD}
               size="small"
               title={t(labelPerformanceGraphAD)}
-              onClick={openModalAnomalyDetection}
+              onClick={(): void => setShowModalAnomalyDetection(true)}
```

Comment:
Can you extract this IconButton inside the renderAdditionalGraphActions?

Reply:
renderAdditionalGraphActions contains the modal that displays the graph and does not contain the action buttons, I think it's not possible :s

Reply:
But it can contains the action buttons too

Reply:
it can t 
![image](https://user-images.githubusercontent.com/97687698/197163753-4ea2362c-35d8-41fd-9af7-f9100960d4cb.png)